models/_sector/nft/trades/ethereum/platforms/looksrare_seaport_ethereum_base_trades.sql

```diff
@@ -0,0 +1,42 @@
+{{ config(
+    schema = 'looksrare_seaport_ethereum',
+    tags = ['dunesql'],
+    alias = alias('base_trades'),
+    partition_by = ['block_date'],
```

Comment:
you can remove this partition for now. rob will be doing a universal approach to partitioning across base models in nft lineage post-migration